protobufs/livekit_analytics.proto

```diff
@@ -202,3 +204,24 @@ message AnalyticsNodeRooms {
   google.protobuf.Timestamp timestamp = 3;
   repeated AnalyticsRoom rooms = 4;
 }
+
+message ReportInfo {
+  oneof message {
+    FeatureUsageInfo feature_usage = 1;
+  }
+}
+
+message FeatureUsageInfo {
+  enum Feature {
+    KRISP_NOISE_CANCELLATION = 0;
+  }
+  Feature feature = 1;
+  string project_id = 2;
+  string room_name = 3;
+  string room_id = 4;
+  string participant_identity = 5;
+  string participant_id = 6;
+  string track_id = 7;
+  google.protobuf.Timestamp started_at = 8;
+  google.protobuf.Timestamp ended_at = 9;
```

Comment:
A couple of notes/questions
1. `AnalyticsEvent` message has a bunch of common fields like `room_id`, `track_id`, but I am making this standalone message fully self-contained. Is that okay?
2. Talking about self-contained, this only has `room_name`, `room_id`, `participant_identity` and `participant_id` and it does not have full `Room` or `ParticipantInfo`. Is that okay? If full objects are needed, will have to get clients to submit it as the controller handling the report will not have all context.

Reply:
this looks good